doc/syslog/syslog-design.md

```diff
@@ -291,26 +362,84 @@ yes - set source IP, no - generate error
 **Example:**
 ```
 Default VRF:
-*.* action(type="omfwd" target="2.2.2.2" protocol="udp" address="1.1.1.1")
+*.notice action(type="omfwd" target="2.2.2.2" protocol="udp" address="1.1.1.1")
 MGMT VRF:
-*.* action(type="omfwd" target="2.2.2.2" protocol="udp" address="1.1.1.1" device="mgmt")
+*.notice action(type="omfwd" target="2.2.2.2" protocol="udp" address="1.1.1.1" device="mgmt")
 DATA VRF:
-*.* action(type="omfwd" target="2.2.2.2" protocol="udp" address="1.1.1.1" device="Vrf-Data")
+*.notice action(type="omfwd" target="2.2.2.2" protocol="udp" address="1.1.1.1" device="Vrf-Data")
+```
+
+#### 3.3.4.5. Filter/Trap: `set/set`
+
+Log regex filter
+
+Filter:
+    Include:
+    Compares the log string against the provided POSIX ERE regular expression.
+    yes - sends log string to remote server, no - do not send
+
+    Exclude:
+    Compares the log string against the provided POSIX ERE regular expression.
+    yes - does not send string log to remote server, no - sends
+
+Protocol:
+messages are forwarded via configured protocol
+
+Trap:
+sends logs with severity equal or higher then configured
+
+**Example:**
+```
+Include:
+:msg, ereregex, "include_str*" 
+*.notice action(type="omfwd" Target="4.4.4.4" Port="514" Protocol="tcp" Device="eth0" Template="SONiCFileFormat")
+Exclude:
+:msg, !ereregex, "exclude_str*" 
+*.info action(type="omfwd" Target="4.4.4.5" Port="514" Protocol="udp" Device="eth0" Template="SONiCFileFormat")
 ```
 
-## 2.4 DB schema
+#### 3.3.4.6. Protocol: `set`
 
-### 2.4.1 Config DB
+messages are forwarded via configured protocol
 
-#### 2.4.1.1 SSIP table
+**Example:**
+```
+*.notice action(type="omfwd" Target="5.5.5.5" Port="514" Protocol="tcp" Device="eth0" Template="SONiCFileFormat")
+```
+
+## 3.4. DB schema
+
+### 3.4.1. Config DB
+
+#### 3.4.1.1. Global syslog table
+```abnf
+; define schema for syslog global configuration attributes under SYSLOG_CONFIG|GLOBAL
```

Comment:
Is this config going to affect the local syslog? The patterns applied here are going to suppress logs at local device. Is thet correct?

Reply:
No, it is not. The new configuration presented in this table is only for sending logs to remote syslog server.